<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetLocationWidget</class>
 <widget class="QWidget" name="TargetLocationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TargetLocationWidget</string>
  </property>
  <widget class="QGroupBox" name="videoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>402</width>
      <height>433</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="videoInput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>24</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>42</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>结束</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="drawButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>画框</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="videoProgressSlider">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {  
border: 1px solid #4A708B;  
background: #C0C0C0;  
height: 5px;  
border-radius: 1px;  
padding-left:-1px;  
padding-right:-1px;  
}  
  
QSlider::sub-page:horizontal {  
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,   
    stop:0 #B1B1B1, stop:1 #c4c4c4);  
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,  
    stop: 0 #5DCCFF, stop: 1 #1874CD);  
border: 1px solid #4A708B;  
height: 10px;  
border-radius: 2px;  
}  
  
QSlider::add-page:horizontal {  
background: #575757;  
border: 0px solid #777;  
height: 10px;  
border-radius: 2px;  
}  </string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="frameCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>24</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>0000/9999</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mapBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>601</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>地 图</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>581</width>
      <height>433</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QMapView" name="mapView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>579</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>579</width>
         <height>400</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(95, 95, 95);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="paramEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>27</red>
               <green>39</green>
               <blue>206</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>113</red>
               <green>88</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>27</red>
               <green>39</green>
               <blue>206</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>113</red>
               <green>88</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>83</red>
               <green>67</green>
               <blue>203</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>113</red>
               <green>88</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>216</red>
               <green>216</green>
               <blue>216</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(216, 216, 216);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadPoseButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>63</width>
           <height>32</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>90</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>姿态参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadMapButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>63</width>
           <height>23</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>90</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>加载地图</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="infoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>信息显示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QTextEdit" name="infoEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(208, 208, 208);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="bottomBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>601</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bottom/Resource/bottom/C_601x201.png);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 18pt &quot;方正姚体&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a7a7a7;&quot;&gt;无人机目标跟踪多任务实验演示平台--&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 18pt &quot;方正姚体&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; color:#ec2643;&quot;&gt;单目标定位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QMapView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qmapview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
